src/components/feedback-particle-effect/feedback-text-effects.ts

```diff
@@ -22,37 +22,26 @@ export class FeedbackTextEffects {
   }
 
   private async loadFont() {
-    const fontName = feedbackCustomFonts[lang] || feedbackTextDefault;
-    const fontPath = `${FONT_BASE_PATH}${fontName}.ttf`;
-    this.applyFontToElement(fontName, fontPath);
-  }
-
-  /**
-   * Applies the specified font to the feedback text element and adds the necessary @font-face rule.
-   * @param fontName - The name of the font to apply.
-   * @param fontPath - The path to the font file.
-   */
-  private applyFontToElement(fontName: string, fontPath: string) {
-    if (fontPath) {
-      const style = document.createElement("style");
-      style.textContent = `
-        @font-face {
-          font-family: '${fontName}';
-          src: url('${fontPath}') format('truetype');
-        }
-      `;
-      document.head.appendChild(style);
-    }
+    const fontOptions = {
+      customFonts: feedbackCustomFonts,
+      defaultFont: feedbackTextDefault,
+      lang: lang,
+    };
 
-    if (this.isFeedbackElementAvailable()) {
-      this.feedbackTextElement!.style.fontFamily = `${fontName}, sans-serif`;
-    }
+    applyFontToElement(this.feedbackTextElement, fontOptions, FONT_BASE_PATH);
   }
 
   public wrapText(text: string): void {
     if (!this.isFeedbackElementAvailable()) return;
-
     this.feedbackTextElement.textContent = text;
+
+    // Dynamically adjust the font size based on the length of the text
+    if (text.length >= 12) {
+      this.feedbackTextElement.style.fontSize = "30px";
+    } else {
+      this.feedbackTextElement.style.fontSize = ""; // Reset to default font size (or specify default if needed)
```

Comment:
this can be simplified in one line, try ternary.

Reply:
thanks @bernhardccodev done.